src/main/java/Duke.java

```diff
@@ -78,39 +40,109 @@ public static void markTask(Task[] taskList, String markAction, int taskNumber)
                 } else {
                     targetTask.isDone = true;
                     System.out.println("Nice! I've marked this task as done: ");
-                    System.out.println("   [X] " + targetTask.getTaskDescription());
+                    System.out.println(targetTask.toString());
                 }
             } else { // Unmark the task
                 if (targetTask.isDone == false) {
                     System.out.println("Task is already marked as not done yet.");
                 } else {
                     targetTask.isDone = false;
                     System.out.println("OK, I've marked this task as not done yet: ");
-                    System.out.println("    [ ] " + targetTask.getTaskDescription());
+                    System.out.println(targetTask.toString());
                 }
```

Comment:
perhaps you can consider separating this into 2 methods - markTask and unmarkTask, this will make it neater. Currently, the if-else statement may be a bit messy because both mark and unmark commands are under the same method markTask()